### ISSUE TITLE
Strava inline skating not synced into tapiriik

### ISSUE THREAD

**Issue by luciaschmidt** [cpfair/tapiriik#338]
I'w got several activites with type "inline skate" in strava, but they don't even appear in tapiriik dashboard. When i edit them and change to cycling, then they appear and also sync into other services. I really like to go skating with my suunto watch and would like to have it also in other services (Endomondo), pls pls can you fix this one? 
Thank you for this great project.

**Comment by mtomana** [comment]
Yep, I can confirm the issue. I was googling for the solution and ended up here.
I can provide example training activities, if you want.

**Comment by Virako** [comment]
I have the same problem
